repository/model/setting.go

```diff
@@ -0,0 +1,109 @@
+package model
+
+import (
+	"encoding/json"
+	"fmt"
+
+	svcmodel "release-manager/service/model"
+)
+
+const (
+	organizationNameKey  = "organization_name"
+	defaultReleaseMsgKey = "default_release_msg"
+	slackSettingKey      = "slack_setting"
+	githubSettingKey     = "github_setting"
+)
+
+type Setting struct {
+	Key     string `json:"key"`
+	Setting any    `json:"setting"`
+}
```

Comment:
<img width="524" alt="Snímek obrazovky 2024-04-11 v 17 19 15" src="https://github.com/jan-zabloudil/release-manager/assets/81581922/6a7e8320-1ea0-460e-9ce4-d25fec88ca84">

In the database, each setting is saved in JSON field. This approach simplifies the process of adding new settings.